.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+
+By participating in this project, you agree to abide by its terms.
+
+## How can I help?
+
+Here are some ways you can help along with some guidelines.
+
+### Documentation
+
+As a user of `unform`, you're the perfect candidate to help us improve our documentation!
+
+Typos, errors, lack of examples and/or explanation, etc. This are some examples of for things that could be fixed and/or improved.
+
+You could even make improvements to this guide! :)
+
+While documenting, try to keep things simple and clear.
+
+### Issues
+
+Some issues are created with missing information, without a template, not reproducible, or plain
+invalid.
+
+You can make them easier to understand and resolve.
+
+#### Submitting an issue
+
+- Please search for similar issues before opening a new one;
+- Use one of the corresponding issue template;
+- Use a clear and descriptive title;
+- Include as much information as possible by filling out the provided issue
+  template;
+- A lot of times, the best way to report an issue is a failing test proving it.
+
+### Feedback
+
+The more feedback the better! We're always looking for more suggestions and opinions on discussions. That's a good opportunity to influence the future direction of this tool.
+
+This include submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality.
+
+The
+[`question` label](https://github.com/Rocketseat/unform/labels/question)
+is a good place to find ongoing discussions.
+
+### Code
+
+You can use issue labels to discover issues you could help out with:
+
+- [`bug` issues](https://github.com/Rocketseat/unform/labels/bug)
+  are known bugs we'd like to fix;
+- [`enhancement` issues](https://github.com/Rocketseat/unform/labels/enhancement)
+  are features we're open to including.
+
+The
+[`help wanted`](https://github.com/Rocketseat/unform/labels/help%20wanted)
+and
+[`good first issue`](https://github.com/Rocketseat/unform/labels/good%20first%20issue)
+labels are especially useful.
+
+You may find an issue is assigned. Please double-check before starting on this issue because somebody else is likely already working on it.
+
+#### Dev environment
+
+When developing, prefer using **Node** ≥ 8 and **yarn**. Writing code with the latest stable Node versions allows us to use newer developer tools.
+
+After [cloning the repository](https://help.github.com/articles/cloning-a-repository/), run `yarn` to install dependencies.
+
+A summary of the scripts:
+
+- To run the example use `yarn dev:example`;
+- The `yarn dev:start` command will build the lib and watch files in bundle and rebuild on changes;
+- Use `yarn test` to run the test suite (powered by [Jest](https://facebook.github.io/jest/));
+- Run `yarn coveralls` to provide test coverage statistics;
```

Comment:
`yarn coveralls` should not be used in dev environment as the mains purpose of it is sending coverage statistics to coveralls.io, so the `yarn test` command will also return coverage report.

Reply:
I should have tested it before including. 😄

Awww, too bad, I really like seen coverage report locally. I'll remove it.

Reply:
Actually... Should I remove it or make a note about its use?

Reply:
I think that you can keep the note telling user that he won't use it. Code coverage is available through `yarn test` command.